core/integration/features/verified/Funding-treasury.feature

```diff
@@ -220,7 +220,7 @@ Feature: check the insurance pool getting shared equally between all markets wit
       | party | reference | status         |
       | aux   | aux-1-19  | STATUS_STOPPED |
 
-    And the network treasury balance should be "1824" for the asset "USD"
+    And the global insurance pool balance should be "1824" for the asset "USD"
```

Comment:
On line 217 above, can we make the comment say "global insurance pool"?

Reply:
done